embedded-kafka/src/test/java/net/manub/embeddedkafka/ConsumerOpsTest.scala

```diff
@@ -0,0 +1,53 @@
+package net.manub.embeddedkafka
+
+import net.manub.embeddedkafka.ConsumerExtensions._
+import org.apache.kafka.clients.consumer.{ConsumerRecord, ConsumerRecords, KafkaConsumer}
+import org.apache.kafka.common.TopicPartition
+import org.mockito.Mockito.{times, verify, when}
+import org.scalatest.mockito.MockitoSugar
+
+import scala.collection.JavaConverters._
+
+class ConsumerOpsTest extends EmbeddedKafkaSpecSupport with MockitoSugar {
+
+  "ConsumeLazily " should {
+    "retry to get messages with the configured maximum number of attempts when poll fails" in {
+      val consumer: KafkaConsumer[String, String] = mock[KafkaConsumer[String, String]]
+      val consumerRecords = new ConsumerRecords[String, String](mapAsJavaMap(Map.empty))
+
+      when(consumer.poll(1)).thenReturn(consumerRecords)
```

Comment:
maybe extracting `1` with giving a meaningful value name would be better (here and in subsequent places)